### PR TITLE
Привет. На проверку.

### PR DESCRIPTION
**Pull request by userhost1**
homeWork_03
JB_chapter1_ALL

### CODE REVIEW

File: src/homework03/JB_chapter1_04.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+//Определить число, полученное выписыванием в обратном порядке цифр любого 4-х значного натурального числа n.
+
+import java.util.Random;
+
+public class JB_chapter1_04 {
+    public static void main(String[] args) {
+
+        int number = new Random().nextInt(8999) + 1000;
+
+        System.out.println(number);
```
Comment (TheFallen058) [inline src/homework03/JB_chapter1_04.java:12]: System.out.println("Original number: " + number);

File: src/homework03/JB_chapter1_04.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+        String s = String.valueOf(number);
+
+        String newNumber = "";
+
+        for (int i = s.length() - 1; i >= 0; i--) {
+            char c = s.charAt(i);
+            newNumber += c;
+        }
+
```
Comment (TheFallen058) [inline src/homework03/JB_chapter1_04.java:22]: System.out.println("New number: " + newNumber);

File: src/homework03/JB_chapter1_04.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        int i = Integer.parseInt(newNumber);
+
+        System.out.println((char) i);
```
Comment (TheFallen058) [inline src/homework03/JB_chapter1_04.java:25]: Not clear what did you try to print here

File: src/homework03/JB_chapter1_05.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
+package homework03;
+
+//Дано любое натуральное 4-х значное число. Верно ли, что все цифры числа различны?
+
+import java.util.Random;
+
+public class JB_chapter1_05 {
+    public static void main(String[] args) {
```
Comment (TheFallen058) [inline src/homework03/JB_chapter1_05.java:8]: Please read the task more carefully.
You should generate one 4 digit number and check if this digits are different

File: src/homework03/JB_chapter1_06.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+//  Создайте число.
+//  Определите, является ли число трехзначным.
+//  Определите, является ли его последняя цифра семеркой.
+//  Определите, является ли число четным.
+
+public class JB_chapter1_06 {
+    public static void main(String[] args) {
+
+        int number = new Random().nextInt();
```
Comment (TheFallen058) [inline src/homework03/JB_chapter1_06.java:13]: To get more clear and readable numbers from Random you can use 
int number = new Random().nextInt(1000);

File: src/homework03/Task06_evenNumbers.java [hunk 6]
```diff
@@ -0,0 +1,12 @@
+package homework03;
+
+public class Task06_evenNumbers {
+    public static void main(String[] args) {
+//          6)Напишите программу вывода всех четных чисел от 2 до 100 включительно
+        for (int i = 2; i <= 100; i++) {
```
Comment (TheFallen058) [inline src/homework03/Task06_evenNumbers.java:6]: You can use i += 2 as counter increment. So check by if will be not required

File: src/homework03/Task07_summ.java [hunk 7]
```diff
@@ -0,0 +1,16 @@
+package homework03;
+
+public class Task07_summ {
+    public static void main(String[] args) {
+//        7)Напишите программу, определяющую сумму всех нечетных чисел от 1до 99
+
+        int result = 0;
+
+        for (int i = 1; i <= 99; i++) {
```
Comment (TheFallen058) [inline src/homework03/Task07_summ.java:9]: same for counter

**Comment by userhost1** [comment]
all fixed

**Pull request closed without merging**